handlers/handlers.go

```diff
@@ -196,14 +206,14 @@ func (e *Env) ReqHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if valid, err := e.validateToken(authHeaderSlice[1]); !valid {
+	if claims, err := e.validateToken(authHeaderSlice[1]); err != nil {
 		errMsg := "Token invalid"
 		log.Println(errMsg + ": " + err.Error())
 		http.Error(w, errMsg, http.StatusUnauthorized)
 		return
+	} else {
```

Comment:
I think this doesn't need to be an `else` block since the corresponding `if` block returns.

Reply:
Yeah it was pretty much because the `claims` variable is set in the `if` block and I was too lazy to separate it lol